ssas/service/main/main.go

```diff
@@ -1,3 +1,29 @@
+/*
+ Package main System-to-System Authentication Service
+
+ The System-to-System Authentication Service (SSAS) enables one software system to authenticate and authorize another software system. In this model, the Systems act automatically, independent of a human user identity. Human users are involved only to administer the Service, including establishing the identities and privileges of participating systems.
+
+ For more details see our repository readme and Postman tests:
+ - https://github.com/CMSgov/bcda-ssas-app
+ - https://github.com/CMSgov/bcda-ssas-app/tree/master/test/postman_test
+
+ If you have a Client ID and Secret you can use this page to explore the API.  To do this, click the green "Authorize" button below and enter your Client ID and secret in the Basic Authentication username and password boxes.
```

Comment:
not sure how to do this but should we update the username and password boxes text to say Client ID and Client Secret?? Unsure about the level of effort tho

Reply:
We get the functionality by saying we use Basic authentication.  I'm pretty sure we can't update the username/password boxes without mucking around in `go-swagger` source.

It _would_ be really, really nice, though.

Reply:
OK, accomplished with a string search/replace.  Presumably we don't update the `.js` bundles very often.  😄